lighthouse-core/report/styles/report.css

```diff
@@ -834,6 +834,43 @@ body {
   outline: 0;
 }
 
+.report-body__icon {
+  width: 24px;
+  height: 24px;
+  border: none;
+  cursor: pointer;
+  flex: 0 0 auto;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: contain;
+  background-color: transparent;
+  margin-left: 8px;
+}
+
+.runtime-config__icon {
+  width: 18px;
+  height: 18px;
+  background-size: contain;
+  transform: translate(0, 1px);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-color: transparent;
```

Comment:
> Can we start a new perfx.css stylesheet and dump all the perfx stuff in that?

Sure. I will separate all the perf-x related CSS code to another file. But for the particular case, I think runtime config section belongs to the main report not just perf-x.